contrib/internal/httputil/trace_gen.go

```diff
@@ -125,6 +125,10 @@ func wrapResponseWriter(w http.ResponseWriter, span ddtrace.Span) http.ResponseW
 			monitoredResponseWriter
 			http.Hijacker
 		}{mw, hHijacker}
+	default:
```

Comment:
This is generated code. If you want changes, you have to modify the code that generates this file. 

https://github.com/DataDog/dd-trace-go/blob/a71ee32b5e69efa07207a5959df796f157ba410c/contrib/internal/httputil/trace_gen.go#L6